src/System.IO.Compression.Brotli/System/Interop/Library.Windows.cs

```diff
@@ -6,7 +6,7 @@ namespace System.IO.Compression
 {
     internal static partial class Library
     {
-        internal const string BrotliEnc = "brotlienc.dll";
-        internal const string BrotliDec = "brotlidec.dll";
+        internal const string _brotliEncoder = "brotlienc.dll";
```

Comment:
The naming of these constants was correct before. Underscore prefix is for private class fields.